packages/common/src/sites/defaults.ts

```diff
@@ -12,6 +12,38 @@ export const DEFAULT_SITE: Partial<IHubSite> = {
   tags: [],
   typeKeywords: ["Hub Site", "hubSite"],
   catalog: { schemaVersion: 0 },
+  assistant: {
```

Comment:
Wondering if we should put this in for all newly created sites? What's the long-term plan for i18n on the strings in this? could we inject as needed? Not necessarily things to resolve with this PR, but maybe make follow up issues b/c once this gets into live items, it's much harder to change (although there is a schemaVersion, so that makes it easier)